access-control-app/int_test/cmr/access_control/int_test/fixtures.clj

```diff
@@ -93,14 +93,15 @@
      (mock-echo-client/reset (conn-context))
      (mdb/reset (conn-context))
      (ac/reset (conn-context) {:bootstrap-data? true})
-     (e/grant-system-group-permissions-to-admin-group (conn-context) :create :read :update :delete)
```

Comment:
I dont think this is related to CMR-2585.  the GROUP ACL never allowed updated and delete (in ECHO or access-control).  We just were not validating and rejecting it when this only went to mock echo